.github/workflows/releaser.yml

```diff
@@ -0,0 +1,44 @@
+name: Library Releaser
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  releaser:
+    name: build_releaser
+    runs-on: ubuntu-latest
+    env:
+      BUILD_TARGETS: aarch64,armv7,x86_64,i686
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set default toolchain
+        run: rustup default stable
+      - name: Set profile
+        run: rustup set profile minimal
+      - name: Add targets
+        run: rustup target add x86_64-linux-android i686-linux-android aarch64-linux-android armv7-linux-androideabi
+      - name: Build
+        run: ./gradlew build
+      - name: release
+        uses: actions/create-release@v1
```

Comment:
This plugin should be fine, but since it's no longer supported have you investigated any of the other alternative ones they [recommend](https://github.com/actions/create-release#github-action---releases-api)?

Reply:
I initially thought, the original actions by GitHub was stable enough for the usage, hence had used it. But i think we can migrate to https://github.com/softprops/action-gh-release